### DISCUSSION TITLE
About distribution phase in simulation

### DISCUSSION THREAD

**Discussion by Harris728** [Open-Systems-Pharmacology/Forum#1130] [Questions & Problems]
Dear community,
Hope you are doing well. As you can see in the picture. The observed data set shows a distribution phase followed elimination phase after Cmax, but simulation curve doesn't fit it and the predicted t1/2 is shorter. I have tried to adjusted many parameters but they don't work. So could you tell me what parameters should I take into consideration? By the way, the simulation curve got to Cmax a little earlier than observed data. What should I do to slow the absorption?

Thanks,
Harris

![PBPK2](https://user-images.githubusercontent.com/94680472/153720577-d433ad99-4e21-4bef-a84b-14c15657dae5.png)
![PBPK](https://user-images.githubusercontent.com/94680472/153720584-b28d6bab-2cee-495a-89bd-056d34a20f55.jpg)

**Comment by prvmalik** [comment]
You likely need a different partition coefficient algorithm (Schmitt for instance) if you wish to hit that terminal point. I would check how that terminal point is determined with respect to BLQs as true mean value of the population may be lower than it is depicting. 

See previous discussion of mine on the forum for different ways to model a late Tmax. See which is most relevant to your scenario. 

Also if still not getting the half life you wish then you could weight the terminal points more heavily in the estimation (parameter identification tool).

**Reply by Harris728** [reply]
Thank you so much for your help. I'll have a try.
